src/styleeditorwindow.cpp

```diff
@@ -18,7 +18,7 @@ StyleEditorWindow::StyleEditorWindow(QWidget *parent)
 {
     m_ui->setupUi(this);
     this->setWindowTitle(tr("Editor Settings"));
-    this->setWindowFlags(Qt::Window | Qt::WindowStaysOnTopHint);
+    this->setWindowFlags(Qt::Tool | Qt::WindowStaysOnTopHint);
```

Comment:
The existing `Qt::WindowStaysOnTopHint` seems wrong. On Xfce, this results in a window that will stay on top by itself, even if another application is brought to the foreground. We really only want the `StyleEditorWindow` to stay on top of the main Notes window (which is achieved by `Qt::Tool` btw, so that change seems appropriate).

Reply:
On bspwm, only having `Qt:Tool` will make the style editor window appear behind the main window, for some reason :thinking:

Reply:
`Qt::Dialog` makes it behave as expected.